TerrorConsole/Assets/_Main/Scripts/LevelManager/LevelManager.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class LevelManager : Singleton<ILevelManagerSource>, ILevelManagerSource
+    {
+
+        public event Action<LevelState> OnLevelStateChange;
+
+        [SerializeField] private LevelState currentLevelState;
+        
+        public enum LevelState
+        {
+            Play,
+            Pause,
+            InDialogue,
+            Cinematic
+        }
```

Comment:
Move it out of the class, inside the namespace.
This way when used you dont need to write "LevelManager" first.